website/project/model.py

```diff
@@ -860,6 +860,7 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     primary_institution = fields.ForeignField('institution')
 
     affiliated_institutions = fields.ForeignField('institution', list=True)
+    visit = fields.IntegerField(default=0)
```

Comment:
Also, the Jira ticket seems to imply that this should be counting the views on the materials submitted to the conference, not the node containing them.